gender_analysis/common.py

```diff
@@ -111,124 +112,50 @@
 # TODO(elsa): Investigate doctest errors in this file, may be a result of
 # my own system, not actual code errors
 
+def load_csv_to_dict(file_path):
+    """
+    Loads a csv file
+
+    :param file_path: can be a string or Path object
+    :return: a list of strings
+    """
+    if isinstance(file_path, str):
+        file_path = Path(file_path)
 
-class FileLoaderMixin:
-    """ The FileLoaderMixin loads files either locally or
-    remotely from Github (if run from an ipython notebook)
+    file_type = file_path.parts[-1][file_path.parts[-1].rfind('.'):]
```

Comment:
Path has a 'suffix' method that grabs the suffix for you, so no need for this kind of thing anymore. See https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix